base_partner_merge/base_partner_merge.py

```diff
@@ -381,12 +381,15 @@ def _merge(self, cr, uid, partner_ids, dst_partner=None, context=None):
             src_partners = ordered_partners[:-1]
         _logger.info("dst_partner: %s", dst_partner.id)
 
-        call_it = lambda function: function(cr, uid, src_partners,
-                                            dst_partner, context=context)
-
-        call_it(self._update_foreign_keys)
-        call_it(self._update_reference_fields)
-        call_it(self._update_values)
+        self._update_foreign_keys(
+            cr, uid, src_partners, dst_partner, context=context
+        )
```

Comment:
The reason I do it is for symmetry and simpler future diffs.

If the `(` has nothing to the right of it, then I put nothing at the left of `)`
It also saves a character on the argument line.


Additionally, if the argument list got longer than 80 chars, I would do two things, put a comma at the end of the last arg and put the closing parens on its own line:
```python
foo(
    arg1,
    arg2,
    arg3,
    arg4,
    ...
    argN,
)
```

Notice the comma after `argN`, this means that if `argN+1,` is added, the diff only has one line.

```diff
+    argN+1
```

Without the comma or with a closing parens, the diff would be three lines
```diff
-    argN)
+    argN,
+    argN+1)
```